akka-stream/src/main/scala/akka/stream/io/SynchronousFileSource.scala

```diff
@@ -25,8 +25,7 @@ object SynchronousFileSource {
    * It materializes a [[Future]] containing the number of bytes read from the source file upon completion.
    */
   def apply(f: File, chunkSize: Int = DefaultChunkSize): Source[ByteString, Future[Long]] =
-    new Source(new SynchronousFileSource(f, chunkSize, DefaultAttributes, Source.shape("SynchronousFileSource")))
-      .named(DefaultAttributes.nameOption.get)
+    new Source(new SynchronousFileSource(f, chunkSize, DefaultAttributes, Source.shape("SynchronousFileSource")).wrap()) // TO DISCUSS: I had to add wrap() here to make the name available
```

Comment:
so:
`grow` == `compose`
`connect` == `wire` | `fuse` | `join` ?
`wrap` == `nest`

The reason I don't like `grow` is that it is too generic, it doesn't say if it grows veritically or horizontally etc. 
`connect` could work, and there are some options that are more `physical` than connect.
`wrap` has been hard, for me, to, no pun intended, wrap my head around. The wrapping is an effect, but it doesn't say anything about the purpose. `Nest` is perhaps clearer about that?